examples/time_frequency/plot_compute_raw_data_spectrum.py

```diff
@@ -70,3 +71,18 @@
 
 ax.set_title('Four left-temporal magnetometers')
 plt.legend(['Without SSP', 'With SSP', 'SSP + Notch'])
+
+# Alternatively, you may also create PSDs from Raw objects with psd_XXX
+f, ax = plt.subplots()
+psds, freqs = psd_multitaper(raw, low_bias=True, tmin=tmin, tmax=tmax,
```

Comment:
I agree that it should be done in another PR. But fixing the `picks` in the example only hides a more serious problem, which is: why do we allow filtering a subset of channels when the proj is for all the channels? Can't you truncate proj when you filter with a subset? That's why I suggested making the other PR first.

If I apply proj first, then filter (with picks), it is different from filtering (with picks) and then applying proj? Is that correct? What about when there is no picks and you have all channels? Does the order in which you apply proj and filter matter? According to your reasoning, it shouldn't matter because one is temporal and the other is spatial filtering. But with this behavior, I'm confused. Also, we need additional tests for these corner cases.